.github/workflows/appengine-deploy-it.yml

```diff
@@ -10,52 +15,66 @@ jobs:
       with:
         service_account_key: ${{ secrets.APPENGINE_DEPLOY_SA_KEY_JSON }}
         export_default_credentials: true
+    - name: create yaml
+      run: |-
+        cat <<EOF > ${{ github.workspace }}/app.yaml
+        service: "${{ github.job }}-${{ github.run_number }}"
+        runtime: "python37"
+        EOF
     - id: deploy
       name: Deploy to App Engine
       uses: ./appengine-deploy
       with:
         project_id: ${{ secrets.APPENGINE_DEPLOY_PROJECT_ID }}
-        deliverables: ./appengine-deploy/test/app.yaml
+        deliverables: ${{ github.workspace }}/app.yaml
         version: gcloud
     - name: Test Output
-      run: curl '${{ steps.deploy.outputs.url }}'
+      run: curl -sfL '${{ steps.deploy.outputs.url }}'
```

Comment:
Curl doesn't actually exit non-zero by default

Reply:
Does it make sense to add a retry with `--retry-delay`? This deploy looks successful.
https://github.com/GoogleCloudPlatform/github-actions/pull/123/checks?check_run_id=747208477

Reply:
Done